test/listNfts.js

```diff
@@ -1,12 +1,18 @@
-const { expect } = require("chai");
```

Comment:
This test does basically nothing afaict- is `ListNfts` used for anything

Reply:
its just used in frontend as a specialized multicall